tests/foreman/api/test_repository.py

```diff
@@ -2482,46 +2388,19 @@ def test_positive_symlinks_sync(self):
         pass
 
 
+@pytest.mark.skip_if_not_set('container_repo')
 class TestTokenAuthContainerRepository:
     """These test are similar to the ones in ``TestDockerRepository``,
     but test with more container registries and registries that use
     really long (>255 or >1024) tokens for passwords.
-    These test require container registry configs in the
-    container_repo section of robottelo.yaml
 
     :CaseComponent: ContainerManagement-Content
 
-    :Assignee: mzalewsk
+    :Assignee: mshriver
```

Comment:
Who is the actual component owner here?

Reply:
No one has been assigned this, I brought it up to leads and should be getting assigned this week.  So I'm going to take ownership until I've got somebody to assign it to.